### DISCUSSION TITLE
Max Accel to decel on klipper v0.12.0-124

### DISCUSSION THREAD

**Discussion by Winky1st** [Guilouz/Klipper-Flsun-Speeder-Pad#164] [Q&A]
Hi all.
On new klipper update -  max_accel_to_decel has been deprecated.
So, removing that line from printer.cfg>printer settings, it seems working with no alarms for me.
Are there new functions to be added in its place?
Thankyou @Guilouz for help.

wish all a great day - cheers.

**Comment by AbitNutzT00** [comment]
Is this the line you're referring to, to remove?
max_accel: 10000

**Reply by Winky1st** [reply]
nope Bro! 
-> line 77  -->   max_accel_to_decel: 5000
Cheers

**Reply by Guilouz** [reply]
Config files have been updated for that.

**Reply by AbitNutzT00** [reply]
Ok, I was going blind trying to find it.
